components/List/index.js

```diff
@@ -0,0 +1,38 @@
+import { StyledCard } from "../Card/Card.styled";
+import { StyledSpan } from "../Card/Card.styled";
+import Image from "next/image";
+
+export default function List({ rides }) {
+  return (
+    <ul>
+      {rides.map((ride) => (
+        <StyledCard key={ride.id}>
+          <p>
+            <StyledSpan>{ride.driver}</StyledSpan>
+            <StyledSpan>
+              <Image
+                src={"/images/charlesdeluvio-K4mSJ7kc0As-unsplash.jpg"}
+                alt="profile pic"
+                width={100}
+                height={100}
+              />
+            </StyledSpan>
+            <StyledSpan>{ride.rating}</StyledSpan>
+          </p>
+          <p>
+            <StyledSpan>
+              {ride.date}, {ride.time}
+            </StyledSpan>
+            Von: {ride.departure}
+            <StyledSpan>Nach: {ride.destination}</StyledSpan>
+            <StyledSpan>Verfügbare Sitzplätze: {ride.seats}</StyledSpan>
+            <StyledSpan>
+              Preis: {ride.price}
+              {ride.currency}
+            </StyledSpan>
+          </p>
```

Comment:
Maybe you don't need so many span? How about like this
```javascript
            <li key={ride.id}>
            <div>
            <p>{ride.driver}</p>
            <Image
                src={"/images/charlesdeluvio-K4mSJ7kc0As-unsplash.jpg"}
                alt="profile pic"
                width={100}
                height={100}
              />
              <p>{ride.rating}</p>
          </div>
          <div>
            <p>{ride.date}, {ride.time}</p>
            <p>Von: {ride.departure}<p>
            <p>Nach: {ride.destination}</p>
            <p>Verfügbare Sitzplätze: {ride.seats}<p>
            <p> Preis: {ride.price}<p>
            <p>{ride.currency}<p>
          </div>
          </li>
``` 
then style the p, div, li  with different padding oder margin to adjust the distance between them. 
I think, this part can be a component. e.g. Card.

Reply:
@LuYujin9 yeah you are right, it looks a bit over the top with all this `<StyledSpan` but I discussed this with Alex earlier. Could you explain more in detail what is your concrete suggestion? ☺️

Reply:
Maybe a [description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) (`<dl>`) with `dt` and `dd` is a proper semantic element here.

Reply:
Thank you both! Anyways, @PReichetanz can we have a short chat on this? I don't understand how to implement this new code structure.